Library/Homebrew/cmd/update-bash.sh

```diff
@@ -198,45 +243,43 @@ update-bash() {
       --simulate-from-current-branch) HOMEBREW_SIMULATE_FROM_CURRENT_BRANCH=1 ;;
       --*) ;;
       -*v*) HOMEBREW_VERBOSE=1 ;;
-      -*v*) HOMEBREW_DEBUG=1 ;;
+      -*d*) HOMEBREW_DEBUG=1 ;;
```

Comment:
Not sure if you want to address this, but this fails for something like `-vd` because once a case is handled, no other cases are evaluated. My idea for a fix would be to collect all short options in a string (in the `-*)` case) and then after the `for` loop do something like:

```shell
[[ "${short_options}" = *d* ]] && HOMEBREW_DEBUG=1
[[ "${short_options}" = *v* ]] && HOMEBREW_VERBOSE=1
```